builds/e2e/longhaul.yaml

```
@@ -0,0 +1,64 @@
+trigger: none
+
+jobs:
+
+################################################################################
+  - job: LinuxAMD64Moby
```

This is a huge nit, but all of the other job names are in snake_case
```suggestion
  - job: linux_amd64_moby
```

---

I don't know I can use suggestions; love it!